include/ProgramRepresentation/DisjointedPVAliasSets.h

```diff
@@ -0,0 +1,74 @@
+#ifndef DISJOINED_PV_ALIAS_SETS_H
+#define DISJOINED_PV_ALIAS_SETS_H
+
+#include "ProgramRepresentation/PVAliasSet.h"
+
+// manages a disjoint set (https://en.wikipedia.org/wiki/Disjoint-set_data_structure)
+// of program variable must-aliases.
+// each program variable is an element of no more than 1 set.
+class DisjointedPVAliasSets {
+  private:
+    std::list<PVAliasSet> sets;
+
+    // returns a reference to the alias set programVar belongs to. if it's not
+    // found, .end() is returned.
+    typename std::list<PVAliasSet>::iterator findIter(ProgramVariable programVar);
+
+    // puts the elements of set2 into set1 in constant time. set2 is also deleteed
+    // from our set of sets.
```

Comment:
This comment should also indicate how `merge` will behave if either `set1` or `set2` is not present.